client/src/components/layouts/MainLayout.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import {Link} from "react-router-dom";
+import styles from "./MainLayout.module.css";
+import { FiHome, FiTruck, FiUser, FiLogOut } from 'react-icons/fi';
+import { BiNotepad } from 'react-icons/bi';
+import { BsBoxSeam } from 'react-icons/bs';
+import logo from "@assets/Logo_128x128.svg";
+
+function MainLayout({children}) {
+    return (
+        <div className={styles.layout}>
+            <header>
```

Comment:
I would suggest separating the header here into a separate component like <Header />, probably also separate the <Logo /> as well, and wrap the navlinks with a <nav> tag. Just place them in the common module, except for the header.

Additionally, I'd suggest that the navbar component be passed an array of objects that contains the route link, the name of the route and the icon, that way we can reuse it for both manager and employee.